pandas/core/arrays/categorical.py

```diff
@@ -645,7 +645,7 @@ def from_codes(cls, codes, categories=None, ordered=None, dtype=None):
             Whether or not this categorical is treated as an ordered
             categorical. If not given here or in `dtype`, the resulting
             categorical will be unordered.
-        dtype : CategoricalDtype or the string "category", optional
+        dtype : CategoricalDtype or the str "category", optional
```

Comment:
```suggestion
        dtype : CategoricalDtype or {"category"}, optional
```

Reply:
I think for a single option, the {} brackets look a bit strange, I would maybe just do `CategoricalDtype or "category"`